tests/helpers/dataset.py

```diff
@@ -0,0 +1,564 @@
+"""Test Helpers - Dataset."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import shutil
+from contextlib import ContextDecorator
+from pathlib import Path
+from tempfile import mkdtemp
+
+import cv2
+import numpy as np
+from scipy.io import savemat
+from skimage import img_as_ubyte
+from skimage.draw import random_shapes
+from skimage.io import imsave
+
+from anomalib.data import DataFormat
+from anomalib.data.utils import Augmenter, LabelName
+
+
+class DummyImageGenerator:
+    """Dummy image generator.
+
+    Args:
+        image_shape (tuple[int, int], optional): Image shape. Defaults to (256, 256).
+
+    Examples:
+        To generate a normal image, use the ``generate_normal_image`` method.
+        >>> generator = DummyImageGenerator()
+        >>> image, mask = generator.generate_normal_image()
+
+        To generate an abnormal image, use the ``generate_abnormal_image`` method.
+        >>> generator = DummyImageGenerator()
+        >>> image, mask = generator.generate_abnormal_image()
+
+        To generate an image with a specific label, use the ``generate_image`` method.
+        >>> generator = DummyImageGenerator()
+        >>> image, mask = generator.generate_image(label=LabelName.ABNORMAL)
+
+        or,
+        >>> generator = DummyImageGenerator()
+        >>> image, mask = generator.generate_image(label=LabelName.NORMAL)
+    """
+
+    def __init__(self, image_shape: tuple[int, int] = (256, 256)) -> None:
+        self.image_shape = image_shape
+        self.augmenter = Augmenter()
+
+    def generate_normal_image(self) -> tuple[np.ndarray, np.ndarray]:
+        """Generate a normal image."""
+        image = random_shapes(image_shape=self.image_shape, min_size=256, max_shapes=1, shape="rectangle")[0]
```

Comment:
If it's just a full rectangle do we need `random_shapes`? Maybe we can do something like
```python
image = = np.zeros([*self.image_shape, 3])
image[...] = np.random.randint(low=0, high=255, size=[3])
```